mixer/pkg/adapter/test/integration.go

```diff
@@ -0,0 +1,270 @@
+package test
```

Comment:
Some docs here would be good. A high level overview of how to call the functions.